Content.Server/Forensics/Systems/ForensicsSystem.cs

```diff
@@ -11,26 +11,45 @@ public sealed class ForensicsSystem : EntitySystem
         public override void Initialize()
         {
             SubscribeLocalEvent<FingerprintComponent, ContactInteractionEvent>(OnInteract);
-            SubscribeLocalEvent<FingerprintComponent, ComponentInit>(OnInit);
+            SubscribeLocalEvent<FingerprintComponent, ComponentInit>(OnFingeprintInit);
+            SubscribeLocalEvent<DNAComponent, ComponentInit>(OnDNAInit);
         }
 
         private void OnInteract(EntityUid uid, FingerprintComponent component, ContactInteractionEvent args)
         {
             ApplyEvidence(uid, args.Other);
         }
 
-        private void OnInit(EntityUid uid, FingerprintComponent component, ComponentInit args)
+        private void OnFingeprintInit(EntityUid uid, FingerprintComponent component, ComponentInit args)
         {
             component.Fingerprint = GenerateFingerprint();
         }
 
+        private void OnDNAInit(EntityUid uid, DNAComponent component, ComponentInit args)
+        {
+            component.DNA = GenerateDNA();
+        }
+
         private string GenerateFingerprint()
         {
             byte[] fingerprint = new byte[16];
             _random.NextBytes(fingerprint);
             return Convert.ToHexString(fingerprint);
         }
 
+        private string GenerateDNA()
+        {
+            var letters = new List<string> { "A", "C", "G", "T" };
```

Comment:
should probably be race-dependent, maybe have slimes use U not T like bacteria i dunno

not required by any means but would be cool to immediately know the species of a blood puddle (slime is obvious since reagent but others not so much)

Reply:
i can make like the first 2 letters be race-dependent (like slimes = AT, people = AG, etc.) and the other letters would be just random

Reply:
as in the letters pool itself not how its generated
some things use U instead of T (iirc) if using RNA like most bacteria i believe

Reply:
well, I leave it how it is now, maybe someone smarter than me will do it i dunno